packages/sandbox/src/EmbeddedSandbox.ts

```diff
@@ -133,6 +140,10 @@ export class EmbeddedSandbox {
       runtime: this.options.runtime,
       endpoint: this.options.initialEndpoint,
       subscriptionEndpoint: this.options.initialSubscriptionEndpoint,
+      sendOperationHeadersInIntrospection:
+        this.options.sendOperationHeadersInIntrospection == undefined
+          ? true
+          : this.options.sendOperationHeadersInIntrospection,
```

Comment:
Small nit, take it or leave it. What you have is fine as well.
```suggestion
      sendOperationHeadersInIntrospection: sendOperationHeadersInIntrospection: this.options.sendOperationHeadersInIntrospection ?? true,
```

Reply:
but what if `sendOperationHeadersInIntrospection` is false?